.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: [ "develop" ]
+    paths-ignore:
+      - 'docs/**'
+      - 'iOSTestApp/**'
+      - 'tvOSTestApp/**'
+      - 'Example/**'
+      - 'TestApp/**'
+      - 'LICENSE'
+      - '*.md'
+      
+concurrency: 
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: macos-latest
+    environment: KALTURA
```

Comment:
Remove this environment, keep them all as they where and add the user/pass to the main secrets and not inside an environment cause it is needed in other workflows as well.